src/components/UI/Button/ClassCompetitorButton.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import InButtonLabel from './InButtonLabel';
+
+const ClassCompetitorButtonWrapper = styled.div`
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  height: 4.375rem;
+  margin: 1rem;
+  padding: 1rem;
+  color: #323f4b;
+  font-family: 'Mulish', sans-serif;
+  font-size: 1.25em;
+  font-weight: 700;
+  font-style: normal;
+  line-height: 1.5rem;
+  background: white;
+  border: solid 1px #323f4b;
+  border-radius: 0.75rem; ;
+`;
+
+const ClassCompetitorButton = (props) => {
+  const {
+    type,
+    classNumber,
+    classCompleted,
+    contestantsAmount,
+    no,
```

Comment:
This might not be necessary. I would imagine this could be a class object coming in that would have the class number like so: `{classNumber: 0, competitors: [array with length you can check], isCompleted: bool, }` or in case of the competitor: `{competitorId: uuid, numerStartowy: givenNumber, name: johnny, exercises: [array you can use later for storing score maybe, ex32: {isComplete: bool, score: 30}]}`